app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,87 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin {
+  #active_admin_content.left_sidebar, #active_admin_content.collapsed_sidebar {
+    #sidebar {
+      margin-left: 0;
+      display: block;
+    }
+
+    .paginated_collection_contents {
+      clear: none;
+      float: left;
+      width: 100%;
+    }
+
+    .blank_slate_container {
+      clear: none;
+    }
+
+    .columns {
+      clear: none;
+    }
+
+    #main_content_wrapper {
+      float: inherit;
+      width: auto;
+      margin-left: 298px;
+      #main_content {
+        margin: 0;
+        float: inherit;
+        .tabs .comments {
+          .active_admin_comment {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 7<br>Selector `active_admin_comment` should be written in lowercase with hyphens